src/styles/typography.scss

```diff
@@ -0,0 +1,85 @@
+.headline-1,
+.headline-2,
+.subtitle-1,
+.subtitle-2 {
+  font-weight: bold;
+}
+
+.text-body {
+  font-size: 16px;
+  font-weight: normal;
+
+  &.bold,
+  &.bold * {
+    font-weight: bold;
+  }
+}
+
+/* small screen */
+@media only screen and (max-width: 600px) {
```

Comment:
```suggestion
@media #{map-get($display-breakpoints, 'sm-and-down')} {
```

Reply:
Die Vuetify-Variablen funktionieren leider nicht in den scss Files und ich finde gerade nicht heraus, wieso. Ich werde das erst mal abbrechen und mit entsprechendem To Do Kommentar versehen